actionpack/lib/action_controller/renderer.rb

```diff
@@ -71,6 +71,18 @@ def initialize(controller, env, defaults)
     end
 
     # Render templates with any options from ActionController::Base#render_to_string.
+    #
+    # The primary options are:
+    # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
+    # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
```

Comment:
I would focus more on absolute pathing as opposed to explicitness and maybe mention not using this with user input.

Reply:
Ive rendered partials outside of a view. It is a thing people do.

Reply:
I’m fine as written but would love mentioning. That we Rails performs no validation for you and that it shouldn’t be used directly with unsanitized user input.